tests/test_cli.py

```diff
@@ -74,3 +74,18 @@ def test_duplicate_markers(tmpdir, duplicate_marker):
     run(workdir=workdir, targets=["mapped.sorted.tag.mkdup.bam"])
     n_input_fastq_reads = 2 * count_fastq_reads(Path(workdir / "trimmed_barcoded.1.fastq.gz"))
     assert n_input_fastq_reads <= count_bam_alignments(workdir / "mapped.sorted.tag.mkdup.bam")
+
+
+def test_final(tmpdir):
```

Comment:
It’s helpful to use long, explanatory function names in these tests. You could name this `test_final_compressed_reads_exist` or similar. You’ll then also see quicker what a test is supposed to test when pytest tells you the name of it on failure.

Reply:
That is absolutely true! Will change.